jumpcloud-ADMU/Powershell/Start-Migration.ps1

```diff
@@ -799,6 +803,15 @@ Function Test-UserRegistryLoadState {
                 throw "Registry Keys are still loaded after Test-UserRegistryLoadState Testing Exiting..."
             }
         }
+        # If isFolderRedirect is false throw error
+        if (!$isFolderRedirect -and $ValidateFolder) {
```

Comment:
again with the comment above, `$validateFolder` could be unnecessary adding to the complexity of this statement, IMO it should just check that the user directory is "redirected" or "not"